playbooks/roles/ocp-config/README.md

```diff
@@ -31,6 +31,7 @@ Role Variables
 | chronyconfig.allow      | no       | ""             | List of network cidr (X.X.X.X/Y) allowed to sync with bastion configured as NTP server |
 | dhcp_shared_network     | no       |                | Flag to update DHCP server work on a shared network. (Neither ACK nor NACK unknown clients) |
 | cni_network_provider    | no       | OpenshiftSDN   | Sets the default Container Network Interface (CNI) network provider for the cluster |
+| cni_network_mtu         | no       |                | MTU value to assign to the CNI network (Will use NIC MTU if not provided. Recommended values for OpenshiftSDN: <NIC MTU> - 50; OVNKubernetes: <NIC MTU> - 100) |
```

Comment:
Using NIC mtu will break the cluster. The default should be <NIC-MTU - 50> or <<NIC-MTU - 100>
Else can we use 1450 and 1400 respectively which is the current default ?

Reply:
I am not actually setting the NIC MTU. This is what the OCP will default to automatically as it is now.

Reply:
For our projects it is by default `NIC-MTU - 50` since the default is SDN network. May be that statement is confusing, I can remove it if needed.

Reply:
Yeah, since if this playbook is used in a standalone fashion it'll result in non-working cluster. Let's clarify it with example values.